### ISSUE TITLE
Add PRs, Core Reviews, and Issues

### ISSUE THREAD

**Issue by eduncan911** [Shpota/github-activity-generator#9]
Excellent repo, and I've been warning many for years!

However, the activity seems like we only ever commit.  That's not corect.  We also have PRs code reviews, and issues.

Could we add these?  At the very least, we can do PRs and Issues as an API is available for those.

**Comment by Shpota** [comment]
Thank you for the feedback. It is an interesting idea. I'll try to implement it once I have time.

**Comment by eduncan911** [comment]
Yeah, what's funny is I had about equal PRs and Commits before I ran this, as I always squashed my branches before opening PRs.

But now...

![Screenshot_20200802-152752](https://user-images.githubusercontent.com/104980/89130692-eea27180-d4d4-11ea-8993-077238f6d902.png)

**Comment by eduncan911** [comment]
I'm using a modified version of the open PR for the additional options (made `update` work; changed the paths to allow `contribute.py` to work from within the same repo, so I can check in `contribute.py` alongside all of my comments, removed the README updates as 20,000 commits over 10+ years made that an insanely large file, etc).

I'm looking into the PRs, Reviews, and Issues and it looks like there are two options, both of which need an addition package or tool installed:

* Github has released an official CLI tool recently (system package)
* PyGithub python package to import

I'm tinkering with the latter.

**Comment by imkrunal** [comment]
Sadly you can't generate the prs and issues in past using github api or any other way. To solve this i have created a simple nodejs script that you can host on any server and it will create commits, issues and pr requsts using the GitHub apis. Though it can't generate the data in past but if you can keep it running it will generate the data over time.

Here is the repo if anyone wants to use: [Github Activity Creator](https://github.com/imkrunal/github-activity-creator)
